README.md

```diff
@@ -46,6 +49,97 @@ In your template you can do
 {{user.twitter}}
 ```
 and it will populate once the full user is loaded
+
+## How to use
+If you haven't customized any of your adapters, serializers or store, the addon will work out of the box. By default is uses JSONAPISerializer and JSONAPIAdapter.
+
+If you made any customization, you will need to include proper mixins. For adapter:
```

Comment:
You could also keep having it work by default by looking up the application adapter at runtime